schemas/context/search.schema.json

```diff
@@ -39,6 +39,16 @@
           "title": "Page Depth",
           "type": "integer",
           "description": "The page depth in the search results."
+        },
+        "xdm:slot": {
+          "title": "Page Slot",
+          "type": "string",
+          "description": "The section of the page where the search result appeared. For eg. Top, Side"
```

Comment:
Is it possible to mane this an enumeration, even just a soft enum (just meta:enum) that acts like a set of recommendations?